Gemfile

```diff
@@ -100,11 +100,13 @@ gem "raddocs", github: "CruGlobal/raddocs"
 # relevant issue: https://github.com/redis/redis-rb/issues/1142
 gem "redis", "< 5"
 gem "rest-client", "~> 2.1.0"
+gem "crowdin-api", "~> 1.12.0"
 gem "rollbar"
 gem "rubyzip", ">= 1.2.2"
 gem "sidekiq-failures"
 gem "sidekiq-unique-jobs"
 gem "validates_email_format_of"
+gem "CFPropertyList"
```

Comment:
Why?

Reply:
Oh, yes, that can be removed now -- it was used back when I needed to use xml before Daniel told me about crowdin-json